### PR TITLE
S2T3 navbar back function framework and SCSS

### PR DESCRIPTION
**Pull request by HadertS**
Needs to use the state stuff from S2T1 but otherwise complete

### CODE REVIEW

**Comment by HadertS** [comment]
State functionality added from s2t1, now ready for review.

File: src/Nav/index.scss [hunk 1]
```diff
@@ -1,6 +1,20 @@
 .navbar-brand span{
   color:#1aa1b8;
 }
+.navbar button{
+  all:unset;
+  color:#487bff;
```
Comment (alreuben) [inline src/Nav/index.scss]: change colour to $secondary-color variable - also add space after ':'

File: src/Nav/index.scss [hunk 2]
```diff
@@ -1,6 +1,20 @@
 .navbar-brand span{
```
Comment (alreuben) [inline src/Nav/index.scss]: change name of file to styles.scss

File: src/Nav/index.scss [hunk 3]
```diff
@@ -1,6 +1,20 @@
...
+
+  &:focus {
+    outline: auto 0.5rem;
+  }
+  &:hover {
+    outline: auto 0.5rem;
+  }
+}
+
 .navbar{
```
Comment (alreuben) [inline src/Nav/index.scss]: add space before {

File: src/Nav/index.js [hunk 4]
```diff
@@ -1,8 +1,13 @@
...
+    const back = () => {
+        setRestaurantId(null)
+    }
+
     return (
-        <nav className='navbar justify-content-center navbar-expand-lg justify-content-lg-start navbar-light bg-light'>
-            <h4 className='navbar-brand my-auto mx-4'><span>Food</span>Delivery</h4>
+        <nav className='navbar justify-content-center justify-content-lg-between flex-column flex-lg-row navbar-light bg-light px-4 align-items-center'>
+            <h4 className='navbar-brand'><span>Food</span>Delivery</h4>
+            {restaurantId && <button className='' onClick={back}>&lt;&lt;Change Restaurant</button>}
```
Comment (alreuben) [inline src/Nav/index.js]: did you want to give the button a class? or maybe just remove all together if not

**Review by alreuben — CHANGES_REQUESTED** [review]

**Comment by alreuben** [comment]
looks good - only thing is to change '.navbar button' colour to $secondary-color variable but that doesn't exist in the current version of the _variable file so happy to approve and change post-merge

**Review by alreuben — APPROVED** [review]

**Pull request merged**